### ISSUE TITLE
[Regression] Font character display in IDE broken for Cascadia Code 2102.025 (as shipped by Windows Terminal 1.9.1445.0)

### ISSUE THREAD

**Issue by dannya** [microsoft/cascadia-code#489]
### Cascadia family version

2102.025

### Cascadia family variant(s)

Cascadia Code (the version with ligatures)

### Font file format(s)

Windows Terminal included version (TTF (variable))

### Platform

Windows 10 19043.985

### Other Software

Windows Terminal 1.9.1445.0
IntelliJ IDEA 2019.1.4

### What happened?

After the latest update of Windows Terminal Preview (to version 1.9.1445.0), I've noticed that the Cascadia Code v2102.025 font shipped with it (which I set as my main IDE editor font) is now showing unexpected characters when viewing my files:

Cascadia Code:

![font-cascadia](https://user-images.githubusercontent.com/394687/120341536-dd8de100-c2ee-11eb-8b91-e5184956449e.png)

vs. Consolas:

![font-consolas](https://user-images.githubusercontent.com/394687/120341546-debf0e00-c2ee-11eb-80c1-7714e5db6b25.png)

![win-terminal](https://user-images.githubusercontent.com/394687/120341543-de267780-c2ee-11eb-9893-e224d0199fea.png)

Code shown in above screenshots:

```
# source venv/bin/activate
# cd misc
# clear && python test_normalize_all--performance.py


#
import os
os.environ['IS_ENV_TEST'] = 'True'


import sys
sys.path.append('../')

from server.tool.performance.normalize import normalize_file
```

**Comment by aaronbell** [comment]
I suspect that this is due to postscript naming issues that will be solved by the #482 / #466 PR updates as Jet Brains has a particular way of enumerating fonts.

**Comment by aaronbell** [comment]
Closing as a suspected dupe of #482. Feel free to re-open if the next version of the font doesn't solve this for you.
